src/stitches.config.ts

```diff
@@ -73,29 +73,34 @@ export const {
       gray400: '#B5BBBD',
 
       lightBackground: '#E1E1E1',
-      surfaceGray: '#eef3f4',
+      surfaceGray: '#eef1f4', // Grey/2|Surface
       subtleGray: '#F5F5F5',
       almostWhite: '#fbfbfb',
 
       lightBorder: '#DEDEDE',
       lightGray: '#C3CDCF',
       mediumGray: '#A7B0B4',
-      lightText: '#859296',
       border: '#939EA1',
-      placeholder: '#99A2A5',
       placeholderDark: '#7C898D',
       neutralGrayDark: '#52575C',
 
+      // text colors
       primary: '#5E6F74',
       text: '#516369',
       secondary: '#56606f',
+      lightText: '#859296',
+      placeholder: '#99A2A5',
 
       background: '#DFE7E8',
-      pacificBlue: '#00ABBF',
       selected: '#31A5AC',
       lightBlue: '#84C7CA',
       darkBlue: '#5D9C9F',
-      green: '#29D07E',
+      blue: '#2c91ee', // Blue/12
```

Comment:
What's with the comments?

Reply:
they correspond to color names in `src/ui/colors`, which are imported from Figma. i don't think they should all be included in the theme necessarily, so i'm just adding the comments to note when i've copied one over manually

Reply:
Oh, I thanks for clarifying.